static/js/util/test_utils.js

```diff
@@ -1,7 +1,36 @@
 import TestUtils from 'react-addons-test-utils';
+import sinon from 'sinon';
 
 export const modifyTextField = (field, text) => {
   field.value = text;
   TestUtils.Simulate.change(field);
   TestUtils.Simulate.keyDown(field, {key: "Enter", keyCode: 13, which: 13});
 };
+
+export const localStorageMock = (init = {}) => {
+  let storage = init;
+
+  const getItem = sinon.spy(key => storage[key] || null);
+
+  const setItem = sinon.spy((key, value) => {
+    storage[key] = value || "";
+  });
+
+  const removeItem = sinon.spy(key => {
+    delete storage[key];
+  });
+
+  const reset = () => {
+    [getItem, setItem, removeItem].forEach(func => {
```

Comment:
I tried the following in the nodejs repl:

```js
var sinon = require('sinon')

const sandbox = sinon.sandbox;

const spyFunc = x => x + 2

spyFunc(2) // 4
let spy = sandbox.spy(spyFunc)

spy.called // false

spy(2) // 4

spy.called // true

sandbox.restore()

spy.called // still true

sandbox.reset()

spy.called // still true
```

so it seems like the `restore` and `reset` methods on the sandbox don't clear and reset the spys and mocks created by the sandbox, it looks like it just clears the sandbox out so that new mocks may be added (`sandbox.fakes === []`).